README.md

```diff
@@ -57,3 +57,5 @@ Configuration for DBadvisor is provided by environment. The list of all environm
 | dbadvisor.report.location                    | _temporary folder, see_ `java.io.tmpdir` _system property_ |
 | dbadvisor.report.on-shutdown-hook            | true          |
 | dbadvisor.max-issues-count-per-type          | 100           |
+| dbadvisor.oracle.foreign-key.enabled         | true          |
+| dbadvisor.oracle.foreign-key.threshold       | 1             |
```

Comment:
minor: we should keep this properties together with the rest of "oracle" properties

Reply:
done